torch/nn/_intrinsic/__init__.py

```diff
@@ -1,4 +1 @@
-from torch._ops import ops
-
-fq_per_tensor_affine_forward = ops.quantized.fake_quantize_per_tensor_affine_forward
-fq_per_tensor_affine_backward = ops.quantized.fake_quantize_per_tensor_affine_backward
+from .fake_quantize import FakeQuantize
```

Comment:
do we need to import it in intrinsic? there's already public visible namespace in torch/quantization

Reply:
oh, I just moved this from _intrinsic to quantization, forgot to remove this one.